### PR TITLE
Missing docu about events for Luigi_C_container

### PR DESCRIPTION
**Pull request by JohannesDoberer**

### CODE REVIEW

**Review by walmazacn — APPROVED** [review]
Looks good

File: docs/luigi-compound-container-api.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ meta -->
 
 # Luigi Compound Container API
 
-This document outlines the parameters provided by the Luigi Compound Container. Luigi Compound Container provides the possibility to insert multiple webcomponent-based microfrontends in one container
+This document outlines the parameters provided by the Luigi Compound Container. Luigi Compound Container provides the possibility to insert multiple webcomponent-based microfrontends in one containe.</br>
```
Comment (walmazacn) [inline docs/luigi-compound-container-api.md:19]: There's a type in 'containe.' - should be 'container.'

File: docs/luigi-compound-container-api.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ meta -->
...
+In addition you can use standard `addEventListener` function to react on events emmitted by the luigi compound container. The list of events and their meaning can be found [here](https://github.com/SAP/luigi/blob/main/container/src/constants/communication.ts).
```
Comment (walmazacn) [inline docs/luigi-compound-container-api.md:20]: Perhaps capitalized version would be more suitable? This pattern is used in the sentence above:
'Luigi Compound Container'

File: docs/luigi-container-api.md [hunk 3]
```diff
@@ -16,7 +16,8 @@ meta -->
 
 # Luigi Container API
 
-This document outlines the parameters provided by the Luigi Container.
+This document outlines the parameters provided by the Luigi Container.</br>
+In addition you can use standard `addEventListener` function to react on events emmitted by the luigi container. The list of events and their meaning can be found [here](https://github.com/SAP/luigi/blob/main/container/src/constants/communication.ts).
```
Comment (walmazacn) [inline docs/luigi-container-api.md:20]: Perhaps capitalized version would be more suitable? This pattern is used in the sentence above:
'Luigi Container'

**Review by walmazacn — CHANGES_REQUESTED** [review]
Pls check my comments

**Review by walmazacn — APPROVED** [review]
Looks good

File: docs/luigi-compound-container-api.md (hunk 2 above)
Comment (JohannesDoberer) [reply docs/luigi-compound-container-api.md]: thx

File: docs/luigi-container-api.md (hunk 3 above)
Comment (JohannesDoberer) [reply docs/luigi-container-api.md]: thx

**Pull request merged**